Tasks/BashV3/bash.ts

```diff
@@ -94,14 +101,29 @@ async function run() {
             else {
                 targetFilePath = input_filePath;
             }
+
+            const [processedArgs, telemetry] = processBashEnvVariables(input_arguments)
```

Comment:
>processBashEnvVariables [](http://example.com/codeflow?start=47&length=23)

My thought is if this is telemetry-only (enableTelemetry   && !enableSecureArgs  ), we should catch exceptions from processBashEnvVariables and report in telemetry if exception occurred

if enableSecureArgs, we should not catch exceptions, just let them fall though to normal exception handler (which will hopefully fail the task and report the error)

if neither enableTelemetry  nor enableSecureArgs  , no need to call processBashEnvVariables at all

Reply:
@merlynomsft I updated the logic, now this call executes when any of the FF is enabled, and in case of each some results will applied, e.g. telemetry publishing if telemetry is enabled.

Note: We are throwing any exceptions from the envs processor, only stops processing next part of line in case of blockers